File: TuristickaAgencija.Model/Prevoz.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TuristickaAgencija.Model
{
    public class Prevoz
    {
        public int Id { get; set; } 
        public int FirmaId { get; set; }

        public string TipPrevoza { get; set; }

        public int BrojMjesta { get; set; }

        public float CijenaPoMjestu { get; set; }
    }
}

File: UI/turisticka_agencija_admin/lib/models/Prevoz.dart
import 'package:json_annotation/json_annotation.dart';

part 'Prevoz.g.dart';

@JsonSerializable()
class Prevoz {
    int brojMjesta;
    int cijenaPoMjestu;
    int firmaId;
    int id;
    String tipPrevoza;

    Prevoz({required this.brojMjesta,required this.cijenaPoMjestu,required this.firmaId,required this.id,required this.tipPrevoza});

    factory Prevoz.fromJson(Map<String, dynamic> json) => _$PrevozFromJson(json);

    Map<String, dynamic> toJson() => _$PrevozToJson(this);
}